### ISSUE TITLE
Repetition/modification of code example

### ISSUE THREAD

**Issue by widdowquinn** [swcarpentry/r-novice-gapminder#77]
In `13-dplyr.Rmd` there's a code example that reads:

```
gdp_pop_bycontinents_byyear <- gapminder %>%
    mutate(gdp_billion=gdpPercap*pop/10^9) %>%
    group_by(continent,year) %>%
    summarize(mean_gdpPercap=mean(gdpPercap),
              sd_gdpPercap=sd(gdpPercap),
              mean_pop=mean(pop),
              sd_pop=sd(pop),
              mean_pop=mean(pop),
              sd_pop=sd(pop))
```

In context, this should possibly be:

```
gdp_pop_bycontinents_byyear <- gapminder %>%
    mutate(gdp_billion=gdpPercap*pop/10^9) %>%
    group_by(continent,year) %>%
    summarize(mean_gdpPercap=mean(gdpPercap),
              sd_gdpPercap=sd(gdpPercap),
              mean_pop=mean(pop),
              sd_pop=sd(pop),
              mean_gdp_billion=mean(gdp_billion),
              sd_gdp_billion=sd(gdp_billion))
```

or it might be an unintended duplication, which could be removed.

**Comment by naupaka** [comment]
Hi @widdowquinn - good catch. I just took a look and it seems like the former option you gave (with the mutated column used in the summary) is the intended version (instead of just being a duplication). Would you be willing to open a quick PR to fix that?

**Comment by widdowquinn** [comment]
Yes, certainly. I was just making a note here for the time being, as I'm doing my lesson plan to teach this for next week. I'll get right on it.

**Comment by naupaka** [comment]
Awesome, many thanks for finding the mistake.

**Comment by naupaka** [comment]
Hey @pschloss want to submit your fix in commit c64b38f for this as a PR so I can close this issue?

**Comment by pschloss** [comment]
My bad, I thought I filed the PR way back when, but actually filed the PR to myself 😬

**Comment by naupaka** [comment]
Closed by #103
